product-development/README.md

```diff
@@ -1,5 +1,13 @@
 # Product Development
 
+## Deadlines for major product releases
```

Comment:
(maybe not a discussion for this particular PR but) I still feel like it's useful, for major releases, to have
a. a source of truth for remaining work (i.e. a collection of all the epics we deem blocking -- which is not necessarily the same as the set of epics that are related). This might end up being labels, or a milestone, or a master epic that links all the component epics, or something else.
b. a description of the deliverable, not just as a collection of features but as a user story(s) -- so that when new potential features come up, we have something to check them against ("feature X would be cool, but does it support user story Y?").

Reply:
Thanks @maiamcc for the feedback. Let's practice these two requirements for 1.0, which is the new big release. I've updated the description here for the items you've described. Feel free to comment there if you think it's still missing any other important context. https://app.clubhouse.io/windmill/epic/5149/